dashlord.yml

```
@@ -524,3 +524,7 @@ urls:
   - url: https://zonages.donnees.incubateur.anct.gouv.fr/ui/
     category: startup
     betaId: données-et-territoires
+  - url: https://conseil-refondation.fr
+    category: startup
+
```

```suggestion
    betaId: cnr
```

https://beta.gouv.fr/startups/cnr.html